### PR TITLE
feat: draw mausoleum desirability range

### PR DESCRIPTION
**Pull request by dvincent56**
As requested [here](https://discord.com/channels/738436300876677261/1133288953932423228/threads/1205658610940780595)
radius

### CODE REVIEW

File: src/widget/city_building_ghost.c [hunk 1]
```diff
@@ -419,6 +429,27 @@ static void set_roamer_path(building_type type, int size, const map_tile *tile,
...
+
+    while (desirability_value < 0) {
+        desirability_value += desirability_step_size;
+        negative_range++;
+    }
+
+    city_view_foreach_tile_in_range(tile->grid_offset, building_size, positive_range, city_building_ghost_draw_bonus_range);
+
+    if (type != BUILDING_NYMPHAEUM) {
+        city_view_foreach_tile_in_range(tile->grid_offset, building_size, negative_range, city_building_ghost_draw_malus_range);
```
Comment (crudelios) [inline src/widget/city_building_ghost.c:449]:
```suggestion
        city_view_foreach_tile_in_range(tile->grid_offset, building_size + positive range, negative_range - positive range, city_building_ghost_draw_malus_range);
```

I believe you can make this a little bit more performant by leaving a "hole" to be filled with just the positive range. This needs testing to confirm it works though.

**Review by crudelios — COMMENTED** [review]
Looks good. Besides the only suggestion it only needs updating to resolve the merge conflicts.

File: src/widget/city_building_ghost.c (hunk 1 above)
Comment (dvincent56) [reply src/widget/city_building_ghost.c:449]: it doesn't work

Comment (crudelios) [reply src/widget/city_building_ghost.c:449]: Ah, shame!

**Pull request merged**